tests/java/lang/custom_message_digest_class/testdata/main.java

```diff
@@ -0,0 +1,13 @@
+package crypto;
+
+import java.security.MessageDigest;
+
+// bearer:expected java_lang_custom_message_digest_class
+public class CustomMessageDigest extends MessageDigest {
+  // some custom implementation process
+}
+
```

Comment:
This is not picked up by the pattern because of the `public` presence, but I would expect it to given our other `class` based rule here https://github.com/Bearer/bearer-rules/blob/main/rules/java/shared/lang/instance.yml#L8-L24

Maybe I am missing something? /cc @cfabianski could we look together when you have a moment

Reply:
https://github.com/Bearer/bearer/pull/1478

I just pushed a fix in the branch so that we don't have to wait for a release of the CLI to have a ✅

Reply:
🐻 🙏